oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/MetadataQueryService.java

```diff
@@ -117,4 +97,19 @@ public EndpointInfo getEndpointInfo(final String endpointId) throws IOException
         endpointInfo.setServiceName(serviceIDDefinition.getName());
         return endpointInfo;
     }
+
+    private List<Service> combineServices(List<Service> services) {
+        return new ArrayList<>(services.stream()
+                                                    .peek(service -> {
+                                                        if (service.getGroup() == null) {
+                                                            service.setGroup(Const.EMPTY_STRING);
+                                                        }
+                                                    })
+                                                    .collect(Collectors.toMap(Service::getName, service -> service,
+                                                                              (s1, s2) -> {
+                                                                                  s1.getLayers().addAll(s2.getLayers());
+                                                                                  return s1;
+                                                                              }
+                                                    )).values());
```

Comment:
```suggestion
        return new ArrayList<>(
            services.stream()
                    .peek(service -> {
                        if (service.getGroup() == null) {
                            service.setGroup(Const.EMPTY_STRING);
                        }
                    })
                    .collect(
                        Collectors.toMap(Service::getName, service -> service,
                                         (s1, s2) -> {
                                             s1.getLayers().addAll(s2.getLayers());
                                             return s1;
                                         }
                        )).values());
    };
```

This format seems more readable, although the current one could pass the style check.